### PR TITLE
17764 - Feedback from testing - Tweak Password Option

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:*
https://github.com/bcgov/entity/issues/17764

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/manage-business/SearchBusinessNameRequest.vue [hunk 1]
```diff
@@ -129,6 +129,7 @@ export default class SearchBusinessNameRequest extends Vue {
   @Prop({ default: false }) readonly isGovStaffAccount: boolean
   @Prop({ default: '' }) readonly userFirstName: string
   @Prop({ default: '' }) readonly userLastName: string
+  @Prop({ default: false }) readonly showManageBusinessDialog: boolean
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/SearchBusinessNameRequest.vue:132]: We let this be controlled by EntityManagement now.

File: auth-web/src/components/auth/manage-business/manage-business-dialog/ManageBusinessDialog.vue [hunk 2]
```diff
@@ -738,6 +744,11 @@ export default defineComponent({
       !enableDelegationFeature.value
     })
 
+    const showPasscodeOption = computed(() => {
+      const allowableBusinessPasscodeTypes: string = LaunchDarklyService.getFlag(LDFlags.AllowableBusinessPasscodeTypes) || 'BC,SP,GP'
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/manage-business-dialog/ManageBusinessDialog.vue:748]: BC,SP,GP are the business types when LD isn't working

**Comment by seeker25** [comment]
Still WIP, some stuff not working

**Review by ochiu — APPROVED** [review]
Looks good

**Review by cameron-eyds — APPROVED** [review]
Looks great!

**Pull request merged**